### DISCUSSION TITLE
Where to set a fixed deposit

### DISCUSSION THREAD

**Discussion by kurzer90** [ghostfolio/ghostfolio#2852] [Q&A]
Is there a option to set up a fixed deposit?
Like having 1000USD on a fixed deposit with 3% interest.
Could not found such a activity.

Thanks

**Comment by Avendo07** [comment]
A workaround I found was that, you can add the principal amount as a "Valuable/Item" and then in "Admin Control" -> "Market Data", you can try and map the value for each year and that should probably map it. I haven't tried it myself, and is a workaround. A dedicated feature would be great.

**Comment by huub66** [comment]
Has anyone been successful of setting this up? I've tried the proposed workaround but that's pretty cumbersome. Any tips?
